modules/git/repo_language_stats_gogit.go

```diff
@@ -113,6 +104,16 @@ func (repo *Repository) GetLanguageStats(commitID string) (map[string]int64, err
 
 					sizes[language] += f.Size
 
+					return nil
+				} else if language, has := attrs["gitlab-language"]; has && language != "unspecified" && language != "" {
```

Comment:
Would be nice to handle also to strip everything after and including `?` character:
![attels](https://user-images.githubusercontent.com/165205/142030984-62023426-24c6-48a3-ab62-bbf642b52e62.png)

Reply:
done